assets/Changelog.md

```diff
@@ -125,6 +125,8 @@ Fix bug in USMap dumper with enums with 256 entries - Atenfyr
 ### Lua API
 Improved reliability of `IsValid`
 
+Added `RegisterLoadMapPreHook`/`RegisterLoadMapPostHook` hooks for `UEngine::LoadMap`.
```

Comment:
This is in the wrong section.
It should be in the `New` section at the top, not in the `Changes` section.